Library/Formula/ghostscript.rb

```diff
@@ -27,25 +22,16 @@ class Ghostscript < Formula
     depends_on "autoconf" => :build
     depends_on "automake" => :build
     depends_on "libtool" => :build
-
-    # Uncomment OS X-specific make vars
-    patch do
-      url "https://raw.githubusercontent.com/Homebrew/patches/b8863c4/ghostscript/gs.patch"
-      sha256 "b3c8903c00428f1a065ceda04e3377c3a110ec21bc149547615bc2166cde6163"
-    end
   end
 
+  patch :DATA # Uncomment OS X-specific make vars
+
   option "with-djvu", "Build drivers for DjVU file format"
 
   depends_on "pkg-config" => :build
-  depends_on "jpeg"
-  depends_on "libtiff"
-  depends_on "jbig2dec"
-  depends_on "little-cms2"
-  depends_on "libpng"
-  depends_on :x11 => ["2.7.2", :optional]
   depends_on "djvulibre" if build.with? "djvu"
-  depends_on "freetype"
+  depends_on "little-cms2"
+  depends_on :x11 => :optional
```

Comment:
The 2.7.2 requirement was from almost three years ago (#16924) and I'm guessing it's not necessary?